rental.rb

```diff
@@ -0,0 +1,9 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+  end
```

Comment:
- [ ] Please modify the constructor of `Rental` so `Book` and `Person` are set in it to break the many-to-many relationship. Kindly see this [article](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship) if you need a refresher 🙏 
```ruby
def initialize(date, book, person)
    .
    .

  @book = book
  book.rentals << self

  @person = person
  person.rentals << self
end
```